recipes/cminpack/all/conanfile.py

```diff
@@ -0,0 +1,109 @@
+from conan.tools.cmake import CMake, CMakeToolchain, CMakeDeps, cmake_layout
+from conan.tools import files
+from conan import ConanFile
+import os
+import textwrap
+
+required_conan_version = ">=1.45.0"
+
+
+class CMinpackConan(ConanFile):
+    name = "cminpack"
+    url = "https://github.com/conan-io/conan-center-index"
+    description = "About A C/C++ rewrite of the MINPACK software (originally in FORTRAN)" \
+                  "for solving nonlinear equations and nonlinear least squares problems"
+    topics = ("nonlinear", "solver")
+    homepage = "http://devernay.free.fr/hacks/cminpack/"
+    license = "BSD-like"
+
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+    }
+
+    def generate(self):
+        tc = CMakeToolchain(self)
+        tc.variables["BUILD_EXAMPLES"] = "OFF"
+        tc.variables["CMINPACK_LIB_INSTALL_DIR"] = "lib"
+        tc.generate()
+        deps = CMakeDeps(self)
+        deps.generate()
```

Comment:
```suggestion
```
Not needed, because this recipe has no other dependencies.

Reply:
True, will do